src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -112,6 +134,16 @@ private ImageLoadingTransformer(IHost host, ModelLoadContext ctx)
             // int: id of image folder
 
             ImageFolder = ctx.LoadStringOrNull();
+
+            if (ctx.LoadStringOrNull().Equals("True"))
+            {
+                Type = true; // It is a VBuffer<byte> type
+            }
+            else
+            {
+                Type = false; // It is a ImageDataViewType
+            }
```

Comment:
for single lines please do not use "{"braces

Reply:
Changed